redirects.txt

```diff
@@ -293,3 +293,5 @@ purchase/replenishment/flows/compute_date.rst applications/inventory_and_mrp/inv
 purchase/replenishment/flows/purchase_triggering.rst applications/inventory_and_mrp/purchase/products/reordering.rst				# (#814)
 purchase/replenishment/flows/setup_stock_rule.rst applications/inventory_and_mrp/purchase/products/reordering.rst					# (#814)
 purchase/replenishment/multicompany/setup.rst applications/general/multi_companies/manage_multi_companies.rst	# (#814)
+
+applications/sales/crm/acquire_leads/generate_from_email.rst applications/sales/crm/acquire_leads/generate_from_website.rst       # (#986)
```

Comment:
I recommend moving this line a bit higher up in this file otherwise you will have a conflict in each forward-port PR starting from 15.0.

Reply:
Moved to right under the `# Redirections introduced in 14.0 :` heading.
I couldn't find any other redirections that contained the `applications/sales/crm/acquire_leads/` file path so hoping there's no conflict?

If I did this incorrectly please indicate a more optimal placement 🙏